code/modules/gear_presets/survivors/survivors.dm

```diff
@@ -18,6 +18,74 @@
 
 	var/survivor_variant = CIVILIAN_SURVIVOR
 
+	dress_under = list(
```

Comment:
intentional that all survs'll have access to this?

Reply:
yeap